tests-functional/tests/test_contact_request.py

```diff
@@ -0,0 +1,99 @@
+from time import sleep
+from uuid import uuid4
+import pytest
+from test_cases import OneToOneMessageTestCase
+from constants import DEFAULT_DISPLAY_NAME
+from clients.signals import SignalType
+
+@pytest.mark.rpc
+class TestContactRequests(OneToOneMessageTestCase):
+
+
+    @pytest.mark.dependency(name="test_contact_request_baseline")
+    def test_contact_request_baseline(self, contact_request_count=1):
+
+        await_signals = [
+            SignalType.MESSAGES_NEW.value,
+            SignalType.MESSAGE_DELIVERED.value,
+        ]
+
+        for i in range(contact_request_count):
+
+            message_text = f"test_contact_request_{i+1}_{uuid4()}"
+
+            sender = self.initialize_backend(await_signals=await_signals)
+            receiver  = self.initialize_backend(await_signals=await_signals)
+
+            pk_sender = sender.get_pubkey(DEFAULT_DISPLAY_NAME)
+            pk_receiver = receiver.get_pubkey(DEFAULT_DISPLAY_NAME)
+
+            existing_contacts = receiver.get_accounts()
```

Comment:
Hmm, `accounts_getAccounts` has nothing to do with contacts.
It returns a list of wallet accounts for the logged in user.

For list of contacts use `wakuext_contacts`:
https://github.com/status-im/status-go/blob/50933aa3286e54bfd407f109ce21eabf1417bae7/services/ext/api.go#L359-L361